fairlearn/metrics/metrics_engine.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+from sklearn.metrics import recall_score
```

Comment:
Still keen on getting rid of this.... do we really need to provide all the one line wrappers ourselves?